config/jobs/build-drivers/build-new-almalinux.yaml

```diff
@@ -14,8 +14,6 @@ presubmits:
         - /workspace/build-drivers.sh
         - almalinux
         env:
-        - name: AWS_REGION
```

Comment:
Do we want to retain this? Right now `dbg-go` specifies the region programmatically.

Reply:
@FedeDP does `dbg-go` set by default _eu-west-1_ region?